### ISSUE TITLE
BEE tokens are transferable 

### ISSUE THREAD

**Issue by yeqbfgxjiq** [1Hive/deployment-log#13]
As @pythonpete32 pointed out [here](https://github.com/FEMBusinessModelsRing/web3_revenue_primitives/pull/60), BEE tokens are supposed to be non-transferable, but the current Rinkeby DAO actually marks them as transferable: https://rinkeby.aragon.org/#/0xe520428C232F6Da6f694b121181f907931fD2211/0x3c16dc46b84a6647f8375235ca88dad2c27edb8b

When we move to mainnet and/or build a new DAO should make sure that BEE tokens are non-transferable

**Comment by lkngtn** [comment]
Yup this was due to a bug in the CLI at the time we deployed. Will definitely be fixed when we migrate.
